client/question/components/PrivacyModalDirective.js

```diff
@@ -111,9 +111,9 @@ tie.directive('privacyModal', [function() {
       </style>
     `,
     controller: ['$scope', '$window', 'CookieStorageService',
-      'ServerHandlerService', 'MENU_PAGE_URL_FROM_QUESTION_PAGE',
+      'ServerHandlerService', 'CLIENT_VERSION_HOMEPAGE_URL',
```

Comment:
Maybe call it the GITHUB_HOMEPAGE_URL or GITHUB_PAGE_URL since that's more unambiguous (I think of "client version URL" as the homepage of the standalone version, i.e. /question or perhaps /menu).

Also maybe this stuff should be a separate PR?

Reply:
yeah, probably, sorry, I think I must have grabbed it in an extra commit.